examples/rapier.rs

```diff
@@ -49,10 +55,16 @@ fn setup(
         transform: Transform::from_xyz(4.0, 8.0, 4.0),
         ..Default::default()
     });
+
+    // camera
     commands
-        .spawn_bundle(PerspectiveCameraBundle {
+        .spawn_bundle(Camera3dBundle {
             transform: Transform::from_xyz(-2.0, 2.5, 5.0).looking_at(Vec3::ZERO, Vec3::Y),
+            // projection: bevy::render::camera::Projection::Orthographic(OrthographicProjection {
+            //     scale: 0.01,
+            //     ..Default::default()
+            // }),
```

Comment:
Forgotten cleanup?

Reply:
I left this here to make it easy to test ortho vs. perspective. Should at least leave a comment to that effect.

Reply:
Ah yeah, I see that now